.github/pull_request_template/hotfix.md

```diff
@@ -0,0 +1,27 @@
+# 🚨 Hotfix
+
+## Checklist
+- [ ] Critical issue: #___
+- [ ] Maintainer approval obtained
+- [ ] Branch: `hotfix-[issue]-[desc]`
```

Comment:
We have no branch naming policy that suggest to use `hotfix` at the moment: https://rucio.github.io/documentation/contributing/#3-create-a-local-working-branch